sys-power/tlp/tlp-1.5.0.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit udev optfeature
+
+DESCRIPTION="Optimize laptop battery life"
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+LICENSE="GPL-2"
```

```suggestion

LICENSE="GPL-2"
```

Add a space between DESCRIPTION< HOMEPAGE, SRC_URI, S and LICENSE. See skel.ebuild.